core/services/blockhashstore/delegate.go

```diff
@@ -13,7 +13,7 @@ import (
 	v1 "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/solidity_vrf_coordinator_interface"
```

Comment:
as of now, we don't have a way to specify multiple coordinator v2 plus addresses. After introducing `v2_6`, we will need BHS feeder to monitor both `2_5` and 2_6` until `2_5` is deprecated

Reply:
similar story for vrf jobs.

Reply:
We could just create new BHS jobs w/ the new coordinator address right?